iceberg/iceberg-catalog/src/test/java/org/apache/iceberg/hive/HiveTableTest.java

```diff
@@ -531,7 +531,6 @@ public void testEngineHiveEnabledConfig() throws TException {
     catalog.dropTable(TABLE_IDENTIFIER);
 
     // Enable by hive-conf
-    catalog.getConf().set(ConfigProperties.ENGINE_HIVE_ENABLED, "true");
```

Comment:
This line should have been kept, as it exercises a code path that still exists.
As it is, `hiveEngineEnabled` still returns true but via a different code path, and the preceding comment is misleading.